[2021-03-30 22:32:29,050] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: CMAPS_SNOWFLAKE_INGESTION.DataExtraction 2021-03-31T02:32:07+00:00 [queued]>
[2021-03-30 22:32:29,279] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: CMAPS_SNOWFLAKE_INGESTION.DataExtraction 2021-03-31T02:32:07+00:00 [queued]>
[2021-03-30 22:32:29,280] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-30 22:32:29,280] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-03-30 22:32:29,280] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-30 22:32:29,387] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): DataExtraction> on 2021-03-31T02:32:07+00:00
[2021-03-30 22:32:29,406] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'CMAPS_SNOWFLAKE_INGESTION', 'DataExtraction', '2021-03-31T02:32:07+00:00', '--job-id', '29', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/cmaps_pipeline.py', '--cfg-path', '/var/folders/j6/755w2l7n4x1c9313788klj880000gq/T/tmpf_m5lzvw', '--error-file', '/var/folders/j6/755w2l7n4x1c9313788klj880000gq/T/tmpowye3cia']
[2021-03-30 22:32:29,409] {standard_task_runner.py:77} INFO - Job 29: Subtask DataExtraction
[2021-03-30 22:32:29,393] {standard_task_runner.py:52} INFO - Started process 26236 to run task
[2021-03-30 22:32:29,521] {logging_mixin.py:104} INFO - Running <TaskInstance: CMAPS_SNOWFLAKE_INGESTION.DataExtraction 2021-03-31T02:32:07+00:00 [running]> on host 152.1.168.192.in-addr.arpa
[2021-03-30 22:32:29,729] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=CMAPS_SNOWFLAKE_INGESTION
AIRFLOW_CTX_TASK_ID=DataExtraction
AIRFLOW_CTX_EXECUTION_DATE=2021-03-31T02:32:07+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-03-31T02:32:07+00:00
[2021-03-30 22:32:29,793] {logging_mixin.py:104} WARNING - /Users/harika/Desktop/CSYE_7245/Assignment3/FastAPI/airflow/dags/cmaps_pipeline.py:23 ParserWarning: Falling back to the 'python' engine because the 'c' engine does not support regex separators (separators > 1 char and different from '\s+' are interpreted as regex); you can avoid this warning by specifying engine='python'.
[2021-03-30 22:33:43,561] {logging_mixin.py:104} INFO - Files saved successfully
[2021-03-30 22:33:43,657] {python.py:118} INFO - Done. Returned value was: None
[2021-03-30 22:33:43,842] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=CMAPS_SNOWFLAKE_INGESTION, task_id=DataExtraction, execution_date=20210331T023207, start_date=20210331T023229, end_date=20210331T023343
[2021-03-30 22:33:44,005] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-03-30 22:33:44,090] {local_task_job.py:146} INFO - Task exited with return code 0
